ui/src/treeDataProvider/AzureNode.ts

```diff
@@ -88,8 +88,9 @@ export class AzureNode<T extends IAzureTreeItem = IAzureTreeItem> implements IAz
         await this.treeDataProvider.refresh(this);
     }
 
-    public openInPortal(): void {
-        (<(s: string) => void>opn)(`${this.environment.portalUrl}/${this.tenantId}/#resource${this.id}`);
+    public openInPortal(id?: string): void {
+        id = id === undefined ? id : this.id;
+        (<(s: string) => void>opn)(`${this.environment.portalUrl}/${this.tenantId}/#resource/${id}`);
```

Comment:
Did you intentionally add a slash?:
old: `resource${this.id}`
new: `resource/${id}`

Reply:
No, that was an accident from when I was calling opn twice.  Good catch.